### ISSUE TITLE
Jenkins jobs fail with docker: IPv4 forwarding is disabled

### ISSUE THREAD

**Issue by adborden** [GSA/data.gov#1786]
In this [build](https://ci.sandbox.datagov.us/blue/organizations/jenkins/deploy-ci-platform/detail/deploy-ci-platform/54/pipeline) the docker container failed to build when trying to download pipenv.

```
[2020-06-19T22:41:38.371Z]  ---> [Warning] IPv4 forwarding is disabled. Networking will not work.
```

This is somewhat expected because our hardening roles disable ip forwarding. What is unclear, is why this has never been a problem before.

## How to reproduce

1. Description of steps to reproduce the issue.

**Comment by mogul** [comment]
The referenced PR has unblocked CI. We can further improve this in future, but for now it's not a high priority. We would probably want to fix it before we allow Jenkins to manage production.

**Comment by adborden** [comment]
I opened https://github.com/GSA/ansible-os-ubuntu-16/issues/29 upstream, since that would be the ideal place to address this. What's left on this issue is essentially described there, or if they aren't interested in the change, then adding [3.1.2](https://github.com/GSA/ansible-os-ubuntu-16/blob/master/tasks/section3.yml#L12) to the jenkins.yml playbook.

**Comment by adborden** [comment]
Also, some background in the underlying cause, @FuhuXia ran into a [related issue on staging](https://gsa-tts.slack.com/archives/C2N85536E/p1593038052182900?thread_ts=1593028644.174300&cid=C2N85536E). It looks like with Ansible 2.8, a [bug was fixed in 2.8.4](https://github.com/ansible/ansible/blob/stable-2.8/changelogs/CHANGELOG-v2.8.rst#id22), the sysctl module used to only check and update sysctl.conf, which is applied on boot. Now it will actually test the current system state, so it's more likely to error during the playbook run. https://github.com/ansible/ansible/pull/56153

It's still a little strange that we've never seen this issue pop-up after a reboot, but maybe docker (or an init script) is enabling nat.ipv4.ip_forward and the setting is never disabled once Jenkins starts running jobs. We're only seeing this now because the setting is actually being applied in the middle of an ansible run.

**Comment by nickumia-reisys** [comment]
No more jenkins (and no equivalent issue on cloud.gov)
- https://github.com/GSA/data.gov/issues/3710
